paste/util/field_storage.py

```diff
@@ -0,0 +1,647 @@
+"""FieldStorage class from the deprecated cgi library."""
```

Comment:
I think you need to keep the authorship notes from the source, as well as the license.
I have not checked if it’s even allowed to copy parts of the module, and whether that impacts paste’s license and author notes.

Reply:
Good point. I have added it now. The PSF license is very permissive, It's GPL-compatible, but not infecting.

Reply:
But the license of Python is not just the PSF license

Reply:
My understanding is that the standard library in Python is licensed under the PSF License Agreement and the Zero-Clause BSD license. So I don't see what else (if anything) should be included.

Reply:
Where do you get this impression about the BSD license?!

The license for Python is documented here: https://docs.python.org/3/license.html

Reply:
I mentioned 0BSD only because you said there were more licenses involved. To be more clear, the standard lib itself is not dual licensed, only the examples and doc code are dual licensed PSF and 0BSD, the rest is PSF only. Except some parts which are however not related to the to deprecated code snippets that we want to include.

Is my understanding of the license text that you linked to wrong? Is something still missing in this modified PR?